class2.md

```diff
@@ -0,0 +1,287 @@
+
+**Features/variables:** These are the measurable data that make up our dataset. Some features collected might be useless while others might carry a substantial amount of weight in making the prediction. Since this is a training set one of the variables included is the label that we are trying to predict (`cvd`).
+
+**Inputs:** Features that we have collected and will use to make our prediction. In this dataset our inputs would be all of the columns except `patient_Id` and `cvd`. We would likely remove `pateint_Id` because a random identifier won't have any predictive power and `cvd` would not be included as an input since it is the target we are trying to predict.
+
+**Output:** The variable that we are trying to predict. In this case `cvd`.
+
+### An overview of supervised learning
+
+<p align="center">
+  <img align="middle" width="400" src="images/supervised.png">
+</p>
+
+The goal of supervised machine learning is to fit a model that relates response variables to predictors and can accurately predict the response for future observations of predictor variables. We will cover supervised learning more in depth in class 2 of this series.
+
+Supervised learning is one of the most commonly used forms of machine learning and it's generally easier to implement and interpret than unsupervised learning alogrithms. 
+
+A hallmark of supervised learning algorithms is that they work in two steps: training and testing. This requires training the algorithm on example datasets with inputs that are annotated (or labled) with the correct response. Once training is complete the algorithm will be tested on a seperate labled dataset and it's performance is evaluated.
+
+Ideally an the algorithm will be specific enough to accurately predict the response variable, but generalizable so that it works with new datasets.
+
+There are two subclasses of supervised learning:
+1. Regression: predicting a numeric
+2. Classification: predicting a category
+
+### Some basic examples of supervised machine learning
+
+#### Determining if a tumor is benign or not from an image
+
+<p align="center">
+  <img width="350" alt="" src="images/histopathBreast.jpg">
+</p>
+
+* Input: Images of tumors
+* Output: Binary; begign or malignant
+* Dataset: Need a database of medical images, expert diagnosis of benign or malignant
+
+#### Identifying handwritten letters
+
+<p align="center">
+  <img width="350" alt="" src="images/handwriting.png">
+</p>
+
+* Input: Scanned, handwritten letters
+* Output: The actual character
+* Dataset: Need thousands of handwritten letters and to annotate the correct letter for each one
+
+### How do you train and test a machine?
```

Comment:
I think this class needs to include an example biomedical problem that we can use to frame each of the discussion points in subsequent sections. It doesn't have to be a real example, the one about disease prediction for which you already have a figure below would work. This would be a good topic for us to discuss at our Monday meeting